kfs-core/src/main/resources/edu/arizona/kfs/db/changelog/changesets/db.changelog-UAF-1735.xml

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't understand what all this Liquibase is for. We previously discussed that the KFS 6 DEV/TST DB already contains the DB requirements for this mod:
Julie Wingate·Dec-11 3:17 PM
OK, Role and Role type exist in our upgrade DB:
Role = 1305
Role Type = 1102
Role Type is in table KRIM_TYP_T
Responsibility = 1007
So, it looks like all the DB pieces already exist, the IDs were just modified slightly in the DB upgrade